addon.py

```
@@ -139,38 +140,51 @@ def _handle_client(self, client):
+                            # We must re-encode to bytes because index is in characters
+                            buffer = decoded[index:].encode('utf-8').lstrip()
+
+                            # Execute command in Blender's main thread
+                            def make_execute_wrapper(cmd):
+                                def execute_wrapper():
+                                    try:
+                                        response = self.execute_command(cmd)
+                                        response_json = json.dumps(response)
+                                        try:
+                                            client.sendall(response_json.encode('utf-8'))
+                                        except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)